vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -82,9 +149,73 @@ public void test_deleteTextBlockById_forNonExistentId() throws BlockDoesNotExist
         Optional<ContentBlock> contentBlockOptional = Optional.of(contentBlock);
         when(contentBlockRepository.findById("notARealId")).thenReturn(contentBlockOptional);
         Mockito.doThrow(EmptyResultDataAccessException.class).when(textBlockRepo).deleteById(textBlockId);
+        Mockito.doThrow(EmptyResultDataAccessException.class).when(textBlockRepo).existsById(textBlockId);
         managerToTest.deleteTextBlockById(textBlockId, "slideId_1");
     }
 
+    @Test(expected = BlockDoesNotExistException.class)
+    public void test_deleteSpaceBlockById_forNonExistentId() throws BlockDoesNotExistException {
+        String spaceBlockId = "notARealId";
+        when(contentBlockRepository.findById("notARealId")).thenReturn(Optional.empty());
+        Mockito.doThrow(EmptyResultDataAccessException.class).when(spaceBlockRepo).deleteById(spaceBlockId);
+        Mockito.doThrow(EmptyResultDataAccessException.class).when(textBlockRepo).existsById(spaceBlockId);
+        managerToTest.deleteSpaceBlockById(spaceBlockId, "slideId_1");
+    }
+
+    @Test
+    public void test_deleteSpaceBlock_success() throws BlockDoesNotExistException {
+        String spaceBlockId = "realId";
+        when(contentBlockRepository.findById(spaceBlockId)).thenReturn(Optional.of(contentBlock));
+        managerToTest.deleteSpaceBlockById(spaceBlockId, "slideId_1");
+        Mockito.verify(spaceBlockRepo).deleteById(spaceBlockId);
+
+    }
+
+    @Test
+    public void test_createSpaceBlock_success() {
+        String slideId = "slideId";
+        ISlide slide = null;
+        String title = "this is a space block";
+        Integer contentOrder = 2;
+        Space space = new Space();
+        ISpaceBlock spaceBlock = new SpaceBlock();
+        Mockito.when(slideManager.getSlide(slideId)).thenReturn(slide);
+        Mockito.when(spaceBlockRepo.save((SpaceBlock) spaceBlock)).thenReturn((SpaceBlock) spaceBlock);
+        Mockito.when(spaceBlockFactory.createSpaceBlock(slide, title, (ISpace) space)).thenReturn(spaceBlock);
+        ISpaceBlock createdBlock = managerToTest.createSpaceBlock(slideId, title, contentOrder, space);
+        Assert.assertEquals(createdBlock.getContentOrder(), contentOrder);
```

Comment:
what about the other properties of the space block? They should be asserted as well.